src/corefx/System.Globalization.Native/pal_normalization.h

```diff
@@ -0,0 +1,28 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+//
+
+#include "pal_compiler.h"
+#include "pal_locale.h"
+
+/*
+ * These values should be kept in sync with System.Text.NormalizationForm
+ */
+typedef enum
+{
+    C = 0x1,
```

Comment:
>C [](start = 4, length = 1)

nit: Maybe naming these to FormC, FormD... would be better.

Reply:
again, not something that is diverging from master https://github.com/dotnet/coreclr/blob/1310065f3726f5fe119b7a2f660a77f01104a1ef/src/corefx/System.Globalization.Native/normalization.cpp#L14-L17

Reply:
This is diverging from master. The C++ code in master is using the C++ style `enum class` for this. It means that you qualify the enum value using the enum name, like `NormalizationForm::C`. `C` without qualification does not compile.

You have change it to regular enum. It means that the references to the enum values are just `C`, `D`, etc. and it is not possible to qualify them. The standard naming convention for C enum is to prefix all value with the enum name or something close to it. Take a look at the naming pattern used for C enums in corefx shims.